lib/api/src/store.ts

```diff
@@ -1,20 +1,19 @@
-import { localStorage, sessionStorage } from 'global';
-import { parse, stringify } from 'telejson';
+import store, { StoreAPI } from 'store2';
 
 export const STORAGE_KEY = '@storybook/ui/store';
 
 import { State } from './index';
 
-function get(storage: Storage) {
-  const serialized = storage.getItem(STORAGE_KEY);
-  return serialized ? parse(serialized) : {};
```

Comment:
@ndelangen @tmeasday Do we need `telejson` here?

Reply:
@shilman Added telejson by overriding `set` and `get` functions of the store